### ISSUE TITLE
How to get the GIT version working on Linux?

### ISSUE THREAD

**Issue by mixrat** [armory3d/armory#445]
I downloaded the GIT version of armory from here: [https://github.com/armory3d/armory](https://github.com/armory3d/armory)
In  blender 2.79 I went to user preferences > install Add-on from file > loaded the armory-master/blender/addon/armory.py file

By downloading the [armory-sdk](https://github.com/armory3d/armory_sdk) and defining it's path in the Blender add-on preferences I was able to see added menu items and a play button at the bottom of the view port. I assume the add-on does not include the Armory render engine option though. However, when I pressed the play button I got this error:

```
Running:  ['/home/pc/Documents/BLENDER ADDONS/armory//nodejs/node-linux64', '/home/pc/Documents/BLENDER ADDONS/armory//linux64/resources/app/extensions/kha/Kha/make', 'krom', '-g', 'opengl', '--shaderversion', '330', '--to', 'build_untitled/debug']
Traceback (most recent call last):
  File "/home/pc/Documents/BLENDER ADDONS/armory//armory/blender/arm/props_ui.py", line 557, in execute
    make.play_project(False)
  File "/home/pc/Documents/BLENDER ADDONS/armory//armory/blender/arm/make.py", line 489, in play_project
    state.compileproc = compile_project(target_name='krom')
  File "/home/pc/Documents/BLENDER ADDONS/armory//armory/blender/arm/make.py", line 277, in compile_project
    return subprocess.Popen(cmd)
  File "/home/pc/Desktop/blender-2.79-linux-glibc219-x86_64/2.79/python/lib/python3.5/subprocess.py", line 676, in __init__
    restore_signals, start_new_session)
  File "/home/pc/Desktop/blender-2.79-linux-glibc219-x86_64/2.79/python/lib/python3.5/subprocess.py", line 1282, in _execute_child
    raise child_exception_type(errno_num, err_msg)
FileNotFoundError: [Errno 2] No such file or directory: '/home/pc/Documents/BLENDER ADDONS/armory//nodejs/node-linux64'
```
Fixed this by renaming 'nodejs_bin' to 'nodejs'. But now I get the error:
```
module.js:472
    throw err;
    ^
Error: Cannot find module '/home/pc/Documents/BLENDER ADDONS/armory/linux64/resources/app/extensions/kha/Kha/make'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:423:7)
    at startup (bootstrap_node.js:147:9)
    at bootstrap_node.js:538:3
Finished in 0.14335131645202637
Build failed, check console
```
I don't have the directory 'linux64' in my sdk files. Am I going about this the right way?

**Comment by guzzard** [comment]
[armory](https://github.com/armory3d/armory) is only one part of the Armory SDK. You can get full pre-packaged Armory SDK bundle [here](http://armory3d.org/download.html), I recommend becoming a Paetron.

**Comment by mixrat** [comment]
Then this is a bit misleading: Full source code is available at GitHub.
And [this](https://www.patreon.com/bePatron?u=3705855): 
![selection_006](https://user-images.githubusercontent.com/20234985/34532237-4a0c5974-f11a-11e7-8ade-b6ee08ce0077.png)
The goal was $850/month last week. Now $1,600/month. Will it ever actually be open source?

**Comment by guzzard** [comment]
As far as I know all Armory SDK source code is available here on github. However, as a supporter you get access to a bundle with all the SDK pieces put together, plus a custom Blender build with Armory integration (play directly in the Blender viewport etc). @luboslenco correct me if I'm wrong..

I think this is an awesome project and well worth the cost of a couple of cups of coffee (where I live) per month to support.

**Comment by luboslenco** [comment]
@mixrat, the goal has always been $1600, please do not spread false information.:( Yes, all of the armory sources (which really took years to write) are open to anyone at https://github.com/armory3d.

You can get the 'pre-assembled' armory sdk for free if you wish - after downloading armory through patreon, cancel the pledge before the month ends and you will not get charged.

I am also working on a 'universal' single clone solution at https://github.com/armory3d/armory_sdk for developers - it's not yet finished.

I hope this resolves the issue. Creating an open game engine from scratch and not being licensed to some proprietary entity is not an easy task. If there is a better way to fund in-development engine which is not yet production ready, I am always all ears!

**Comment by mixrat** [comment]
Sorry, I guess I just misread the Patreon info for this project. Maybe the funding was at $850 last week. Blender is a good model for an open source project. They ask for donations, and sell a few products but the 'pre-assembled' version is always freely available. I have donated to Blender and will continue to do so. I would also donate to this project if I found it was a benefit to  my applications. But I cannot know that unless I donate. Beta testers of open source software should not have such an obligation.
